_posts/2024/2024-01-31-why-go-to-a-rails-or-ruby-conference.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Why go to a Ruby or Rails conference?"
+description: "It’ll genuinely make you live longer, amongst other things"
+layout: article
+category: ruby
+date: 2024-01-31 19:00
+image:
+  base: '2024/why-go-to-a-rails-or-ruby-conference'
+  alt: 'The audience at Brighton Ruby 2024'
+  credit: "Clive Andrews"
+  source: "https://cliveandrews.com/"
+---
+
+Conferences have always been a big part of my career as a developer. Attending, speaking and running events—off the side of my desk—have played a massive part in my career, job prospects and personal relationships.
+
+Enough about me. What about you? Why should you take a day (or two, or three!) off work, spend a bunch of money, get lectured from a stage and then have to talk to strangers in a strange town halfway across the country?
+
+## Professional Development
+
+A conference is _not_ training. At most a talk can impart useful syntax, or explain a concept or help to bridge where you are to _where you could be_.
+
+A conference is a great place to find out the latest and greatest thinking in your particular area of work or to be introduced to facets of your career that you haven’t had the chance to consider in the everyday, unrelenting, demand for features and bug fixes.
+
+Even a couple of hours of workshop delivered as part of an event’s programming, _isn’t_ training. You do sometimes get full days of classroom training attached to conferences, but those tend to be specific, full-days & smaller groups; those _are_ training!
+
+If information isn’t being downloaded into your head, then why bother? The videos will be online soon and you can always read the documentation.
+
+Thinking like this misses the point. You won’t watch the videos, the YouTube counts under my own conference talks would indicate those numbers round to zero.
+
+You’re not enabling the serendipity of being in a talk you would have avoided where an insight unpacks a problem that your team have been struggling with for months.
```

Comment:
Struggling with the construction of this sentence. Are you saying "if you don't go you won't be enabling that serendipity.." or "let's face it, that 'magic' moment won't happen"?